File: complete/src/main/java/com/example/springboot/HelloController.java
package com.example.springboot;
import org.springframework.web.bind.annotation.CrossOrigin;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;
@CrossOrigin(origins = "http://localhost:5500")
@RestController
public class HelloController {

  @GetMapping("/")
  public String index() {
    return "Greetings from Spring Boot!";
  }

}

File: complete-kotlin/src/main/kotlin/com/example/springboot/HelloController.kt
package com.example.springboot

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController
ﬁﬁﬁ
@RestController
class HelloController {

  @GetMapping("/")
  fun index(): String = "Greetings from Spring Boot!"
}
